docs/configurations.md

```diff
@@ -0,0 +1,235 @@
+| `scalar.db.password`                 | Either a master or read-only key used to perform authentication for accessing Azure Cosmos DB for NoSQL. |            |
+| `scalar.db.cosmos.metadata.database` | Database name for the namespace and table metadata used for ScalarDB.                                    | `scalardb` |
+
+</div>
+<div id="DynamoDB" class="tabcontent" markdown="1">
+
+The following configurations are available for DynamoDB:
+
+| Name                                  | Description                                                                                                                                                                                                                                                 | Default    |
+|---------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|
+| `scalar.db.storage`                   | `dynamo` must be specified.                                                                                                                                                                                                                                 | -          |
+| `scalar.db.contact_points`            | AWS region with which ScalarDB should communicate (e.g., `us-east-1`).                                                                                                                                                                                      |            |
+| `scalar.db.username`                  | AWS access key used to identify the user interacting with AWS.                                                                                                                                                                                              |            |
+| `scalar.db.password`                  | AWS secret access key used to authenticate the user interacting with AWS.                                                                                                                                                                                   |            |
+| `scalar.db.dynamo.endpoint_override`  | Amazon DynamoDB endpoint with which ScalarDB should communicate. This is primarily used for testing with a local instance instead of an AWS service.                                                                                                        |            |
+| `scalar.db.dynamo.metadata.namespace` | Namespace name for the namespace and table metadata used for ScalarDB.                                                                                                                                                                                      | `scalardb` |
```

Comment:
Since this PR is for `3.8` and earlier, I think this configuration should be as follows, right?

```suggestion
| `scalar.db.dynamo.table_metadata.namespace` | Namespace name for the namespace and table metadata used for ScalarDB.                                                                                                                                                                                      | `scalardb` |
```